internal/ignition/ignition.go

```diff
@@ -1039,8 +1039,6 @@ func sortHosts(hosts []*models.Host) ([]*models.Host, []*models.Host) {
 	workers := []*models.Host{}
 	for i := range hosts {
 		switch {
```

Comment:
Given that now we have just a case and a default, does switch-case still makes sense here?